### PR TITLE
Add jekyll to preview examples

### LINKED ISSUE #3: Update boilerplate theme
- Add gulp jekyll
- Add new run tooling

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Added Jekyll to provide site to showcase examples

## QA

- Pull code
- Run `gulp jekyll`
- View example site at [http://127.0.0.1:4001/vanilla-boilerplate-theme/](http://127.0.0.1:4001/vanilla-boilerplate-theme/)

Fixes #3

### CODE REVIEW

**Comment by nottrobin** [comment]
I don't think `jekyll` is the best name for this. In an ideal world, the user shouldn't have to care at all about what is used to serve the examples, all they care about is that they are served. And we also want to flexibility to swap `jekyll` out for `caddy` or a simply nodejs server or anything else without changing the interface.

So we could call it `serve`, which would make sense.

But another possibility is `start`. The reason for this is that it's an [NPM standard name](https://docs.npmjs.com/cli/start), used [by build.snapcraft.io](https://github.com/canonical-ols/build.snapcraft.io/) for running the server, and so it's the name I've been using in the new `./run` scripts e.g. [for www.ubuntu.com](https://github.com/canonical-websites/www.ubuntu.com).

What do you think @barrymcgee?

**Comment by barrymcgee** [comment]
I'd prefer serve but given start is already being used, best to stick with that?

**Comment by nottrobin** [comment]
@barrymcgee yep sounds good, let's change it to `start` for now. But I think maybe we should discuss `start` vs `serve` in dev catch-up before we get too far down the line. At this stage we could change it before it people get too used to it.

**Comment by barrymcgee** [comment]
@nottrobin This is ready for re-review 👀

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "devDependencies": {
     "es6-promise": "^4.0.5",
     "gulp-autoprefixer": "3.1.1",
-    "gulp-cache": "0.4.5",
+    "gulp-cache": "0.4.X",
```
Comment (nottrobin) [inline package.json]: Why is this 'X'? That doesn't seem like a valid number.

**Pull request merged**